extensions/typescript-language-features/package.json

```diff
@@ -270,6 +270,100 @@
           "description": "%configuration.suggest.includeAutomaticOptionalChainCompletions%",
           "scope": "resource"
         },
+        "typescript.inlayHints.includeInlayParameterNameHints": {
+          "type": "string",
+          "enum": [
+            "none",
+            "literals",
+            "all"
+          ],
```

Comment:
I think we should add descriptions for these too.

- **None**: Disable parameter name hints.
- **Literals**: Enable parameter name hints only for literal arguments.
- **All**: Enable parameter name hints for literal and non-literal arguments.

@DanielRosenwasser, can you think of a more approachable way to describe this?

Reply:
I guess if we can’t find a concise way to explain what literal expressions are or are not, these descriptions don’t add much.

Reply:
Updated.